package.json

```diff
@@ -23,31 +23,29 @@
     "demo"
   ],
   "scripts": {
-    "build": "tsc --project tsconfig.build.json",
+    "build": "npm run clean && tsc --project tsconfig.build.json",
     "clean": "rimraf ./build",
     "demo": "npm run build && npm run --workspace=demo storybook:axe --",
     "demo:axeOnly": "npm run build && npm run --workspace=demo storybook:axeOnly --",
     "demo:build": "npm run --workspace=demo build-storybook",
     "demo:eds": "axe-storybook --storybook-url https://chanzuckerberg.github.io/edu-design-system",
     "demo:storybook": "npm --workspace=demo run storybook",
     "lint": "eslint .",
-    "prebuild": "npm run clean",
     "prepublishOnly": "npm run build",
-    "pretest:integration": "npm run build && npm run demo:build",
```

Comment:
I removed the `pre-foo` scripts, just to make it more clear what's happening when running something like `npm run integration`